development/sass/modules/_hover-card.scss

```diff
@@ -1,4 +1,4 @@
-.blog-preview-list {
+.hover-card {
```

Comment:
Should be in a separate pull request, this is not the hero. I'lll keep repeating until you start listening. Listening is a valuable skill.